### PR TITLE
fix(74): Fix social account

### PR DESCRIPTION
**Pull request by hromogen**
-- Fix the backend part of the social account functionality

-- Refactor user profile page

-- Add hide info message after 2 seconds

### CODE REVIEW

File: packages/server/src/db/db.service.ts [hunk 1]
```diff
@@ -196,27 +201,40 @@ export default class DBService {
...
     description: string,
-    pictureUrl: string,
+    pictureUrl: string
   ) {
-    return this.shelfRepository.update(id, { name, location, description, pictureUrl });
+    return this.shelfRepository.update(id, {
+      name,
+      location,
+      description,
+      pictureUrl,
```
Comment (dartwlad) [inline packages/server/src/db/db.service.ts:236]: I would suggest removing coma after the last property for consistency in whole PR.

Comment (hromogen) [reply packages/server/src/db/db.service.ts:236]: Well, haven't we agreed on the trailing comma es-rule being set to `true` for objects?

Comment (karolinayanko) [reply packages/server/src/db/db.service.ts:236]: we've agreed that we allow to use trailing comma but it is optional

Comment (hromogen) [reply packages/server/src/db/db.service.ts:236]: So... May I leave it?

**Review by karolinayanko — APPROVED** [review]

File: packages/web/src/components/ManageSocials/FbAuth.test.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    return { container, mockDispatch, getByTestId };
+  };
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  const onResponseAction = 'NOICE_ALL_WENT_WELL';
+  const onErrorAction = 'ALARM_SOME_PROBLEM_HAPPENED';
+  const errorMessage = 'Chuck Norris has been defeated by Bruce Lee';
```
Comment (dubbha) [inline packages/web/src/components/ManageSocials/FbAuth.test.tsx:66]: Why is that an error though? The error would be the other way around!

File: packages/web/src/components/ProfileUserDetails/UserAvatarSection.test.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+describe('components/UserAvatarSection', () => {
+  const firstName = 'Chuck';
+  const lastName = 'Norris';
+  const pictureUrl = 'https://my-pics/selfie-with-big-bang.png';
```
Comment (dubbha) [inline packages/web/src/components/ProfileUserDetails/UserAvatarSection.test.tsx:7]: LOL

File: packages/web/src/components/ManageSocials/FbAuth.test.tsx (hunk 2 above)
Comment (hromogen) [reply packages/web/src/components/ManageSocials/FbAuth.test.tsx:66]: I am not quite sure about the question. But the thing that does happen in this place of the file is mocking of the FB auth button API. And yes, we use it like this

**Review by dubbha — APPROVED** [review]

**Pull request merged**